db/fixtures/ae_datastore/ManageIQ/Deployment/ContainerProvider/System/StateMachine/Deployment.class/__methods__/analyze_deployment.rb

```diff
@@ -1,33 +1,22 @@
 def analyze_deployment
   $evm.log(:info, "********************** #{$evm.root['ae_state']} ***************************")
-  $evm.root['deployment_method'] = $evm.root['automation_task'].automation_request.options[:attrs][:deployment_method]
-
-  if $evm.root['deployment_method'] == "managed_provision"
+  if $evm.root['deployment_method'] == "provision"
```

Comment:
we agreed on "managed_existing", "managed_provision" and "unmanaged", from the logic that the first two share some properties (being managed in manageiq). so my question is if there was a reason for the change?